blueocean-dashboard/src/main/js/LoadingIndicator.js

```diff
@@ -44,7 +44,10 @@ export default {
     },
 
     hide() {
-        loadingCount--;
```

Comment:
If you navigate directly to `/blue/create-pipeline` the loading worm displays by default. I was hiding the indicator on mount, which set the value to -1, which wouldn't actually hide it. Maybe the better fix is for it not to display by default. WDYT?